src/api/hocrrenderer.cpp

```diff
@@ -312,43 +310,37 @@ char* TessBaseAPI::GetHOCRText(ETEXT_DESC* monitor, int page_number) {
         tcnt++;
       }
     } else if (tesseract_->lstm_choice_mode == 2 && choiceMap != nullptr) {
-      for (size_t i = 0; i < choiceMap->size(); i++) {
-        std::vector<std::pair<const char*, float>> timestep =
-            (*choiceMap)[i];
+      for (auto timestep : *choiceMap) {
         if (timestep.size() > 0) {
           hocr_str << "\n       <span class='ocrx_cinfo'"
                    << " id='"
                    << "lstm_choices_" << page_id << "_" << wcnt << "_" << tcnt
                    << "'>";
-          for (size_t j = 0; j < timestep.size(); j++) {
+          for (auto & j : timestep) {
```

Comment:
Are you going to run clang-format on these files, after the PR merge?

Reply:
I'm afraid that will be necessary, because clang-tidy seems to ignore the formatting rules from file `.clang-format`.  Maybe it should be done after the last modernize step. Mixing code changes from modernizers and formatting changes makes reviews much more difficult.